schema.json

```diff
@@ -312,6 +312,10 @@
         "up": {
           "description": "SQL expression for up migration",
           "type": "string"
+        },
+        "onComplete": {
+          "description": "SQL expression will run on complete step (rather than on start)",
+          "type": "boolean"
```

Comment:
This field can be set `"default": false`, then the generated field is just a `bool` rather than a `*bool` making it easier to work with.